components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -520,8 +522,10 @@ private String getAuthorizedUserType(OAuthAuthzReqMessageContext authAuthzReqMes
      * @param tokenReqMessageContext     Token request message context.
      * @return authenticated subject identifier.
      */
-    private String getAuthenticatedSubjectIdentifier(OAuthAuthzReqMessageContext authAuthzReqMessageContext,
-            OAuthTokenReqMessageContext tokenReqMessageContext) throws IdentityOAuth2Exception {
+    private String getAuthenticatedSubjectIdentifier
```

Comment:
Shall we revert this formatting to previous one? This is bit complicated.

Reply:
resolve from https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1750/commits/e9efbfd7f7ecd3bc0013e05e9e784765d88022c9